### DISCUSSION TITLE
Transponder- FBW A320Dev Mod

### DISCUSSION THREAD

**Discussion by jaymo10** [flybywiresim/aircraft#3838] [Help]
just downloaded the updated Dev version- transponder not working- Clicking on every button does nothing..cant find any info about how to .. Need assistance. Thanks.

**Comment by ChruutvoLuzi** [comment]
You need to press CLR once for deleting a single character or twice to completely delete the squawk and enter a new one

**Comment by jaymo10** [comment]
That’s the problem! I cannot CLR with the mouse..😭

**Reply by ChruutvoLuzi** [reply]
Likely a mod conflict then. Remove all mods and try reinstalling the latest dev

**Comment by jaymo10** [comment]
Found the culprit!
The copilot mod!! It worked on earlier Dev mods but not on the latest.. So I removed , wait until the next copilot update. Thanks for help!
